database/data.sql

```diff
@@ -10,7 +10,7 @@
 --      ('Build projects', false),
 --      ('Get a job', false);
 
-insert into "products" ("name", "style", "color", "image", "price")
+insert into "products" ("productId", "name", "style", "color", "image", "price")
```

Comment:
You added a "productId" column, but you don't have any values for it. For products, it should be OK to just let the database give them a number, so you can revert this line to what it was before (that is, remove "productId" from the list).